doc/DeveloperGuide.md

```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should take at most 1 second to process any command.
+6. Should be able to handle any valid or invalid user input.
+7. Should be easy to use for new users.
```

Comment:
Recall from tutorial why we should not have subjective NFRs. What do you mean by `easy to use`?